test/acceptance_with_python/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
would be great to change this part to:
```
#!/usr/bin/env bash

set -euo pipefail
```
Please set this second setting to be sure that the exit codes will be propagated properly